QModManager/Patching/Patcher.cs

```diff
@@ -16,7 +16,7 @@ internal static string QModBaseDir
         {
             get
             {
-                if (Environment.CurrentDirectory.Contains("system32") && Environment.CurrentDirectory.Contains("Windows"))
+                if (Environment.CurrentDirectory.ToLowerInvariant().Contains("system32") && Environment.CurrentDirectory.ToLowerInvariant().Contains("windows"))
```

Comment:
This hasn't happened in a while now and the check can be removed without issues.